apps/src/netsim/NetSimSendWidget.js

```diff
@@ -54,24 +86,123 @@ NetSimSendWidget.createWithin = function (element, connection) {
 NetSimSendWidget.prototype.bindElements_ = function () {
```

Comment:
Would be good to include a comment about what the return value of this function means.